cluster/gce/win1803/install-logging-agent.psm1

```diff
@@ -49,19 +62,19 @@ function InstallAndStart-LoggingAgent {
       -ArgumentList "/S" `
       -Wait
 
-    # Install additional plugins required to parse container logs.
-  Start-process 'C:\Program Files (x86)\Stackdriver\LoggingAgent\Main\bin\fluent-gem' `
+  # Install additional plugins required to parse container logs.
+  Start-Process "$STACKDRIVER_ROOT\LoggingAgent\Main\bin\fluent-gem" `
```

Comment:
I noticed in some places we've used `$a` and in others `${a}` for expansion. Is there a guideline for when each is appropriate in PowerShell? E.g. in bash it's often recommended to always use `${a}`.

Reply:
Right, the style guide omits the braces around variables in its example code. I'm not sure why I started using the braces all-the-time when writing this, but now I think it will be easier to just remove them where they're not needed.